### PR TITLE
Update README for Set Up MATLAB

### PR DESCRIPTION
**Pull request by mw-hrastega**
Updated the text and examples. When reviewing the examples, please pay attention to the text in the YAML scripts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,69 @@
...
+      - name: Run tests and generate artifacts
+        uses: matlab-actions/run-tests@v0
+        with:
+          test-results-junit: test-results/results.xml
+          code-coverage-cobertura: code-coverage/coverage.xml
 ```
 
-You can use this action with the [Run MATLAB Command](https://github.com/matlab-actions/run-command/) and [Run MATLAB Tests](https://github.com/matlab-actions/run-tests/) actions to easily run MATLAB and Simulink as part of your build pipeline.
+## Set Up MATLAB
+When you define your workflow in the `.github/workflows` directory of your repositoy, you can specify the **Set Up MATLAB** action using the `setup-matlab` key. The action accepts an optional input.
```
Comment (jwpereira) [inline README.md:54]:
```suggestion
When you define your workflow in the `.github/workflows` directory of your repository, you can specify the **Set Up MATLAB** action using the `setup-matlab` key. The action accepts an optional input.
```

Comment (jwpereira) [inline README.md:54]: Like the other two, `setup-matlab` prolly won't be used unqualified without our org/version 🤔

**Review by jwpereira — APPROVED** [review]
This is so much better. Thanks so much @mw-hrastega 🚀

File: README.md (hunk 1 above)
Comment (mw-hrastega) [reply README.md:54]: Fixed.

Comment (mw-hrastega) [reply README.md]: Updated the text :).

File: README.md [hunk 2]
```diff
@@ -1,53 +1,69 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Install MATLAB
+        uses: matlab-actions/setup-matlab@v0
+      - name: Run script
+        uses: matlab-actions/run-command@v0
+        with:
+          command: 'myscript'
```
Comment (mcafaro) [inline README.md:28]: Consider losing the quotes

**Review by mcafaro — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mw-hrastega) [reply README.md]: Done.

**Pull request merged**